tools/bioformats2raw/bf2raw.xml

```diff
@@ -0,0 +1,127 @@
+<tool id="bf2raw" name="bioformats2raw" version="0.0.1">
```

Comment:
According to https://github.com/elixir-europe/biohackathon-projects-2023/blob/main/16/conventions.md, maybe:
```suggestion
<tool id="bf2raw" name="Convert to OME-Zarr" version="0.0.1">
```
or something similar?

Reply:
Done